Gemfile

```diff
@@ -1,6 +1,8 @@
 source 'https://rubygems.org'
 
-ruby "~> #{File.read(File.join(__dir__, '.ruby-version')).strip}"
+# If you update the rails version, please change the targeted
+# version in .rubocop.yml
```

Comment:
this isn't ideal, since the ruby change happens in `.ruby-version`. there may be a better way to do this!

Reply:
`Configuration files are pre-processed using the ERB templating mechanism` so i am gonna see if i can programmatically generate the version values

Reply:
i did it! this concludes this conversation with myself. thanks for rubber ducking github.